app/buyers/views/buyers.py

```diff
@@ -127,6 +127,10 @@ def view_brief_overview(framework_slug, lot_slug, brief_id):
     sections = content.summary(brief)
     delete_requested = True if request.args.get('delete_requested') else False
 
+    content_loader.load_messages(brief['frameworkSlug'], ['urls'])
+    call_off_contract_url = content_loader.get_message(brief['frameworkSlug'], 'urls', 'call_off_contract_url')
```

Comment:
But maybe a quick check to see if the url doesn't exist then does `get_message` return None or does the app fall over?

Will only really make sure we can't put a new framework live without having the correct messages set up....